### DISCUSSION TITLE
Run as another user like ConEmu

### DISCUSSION THREAD

**Discussion by JGomsss** [Eugeny/tabby#5805] [Q&A]
Hi,
I tryed to set a new profil to launch powershell under another identity.
It is possible on ConEmu to start shell and define the user with or without password like this :
`cur_console:u:"Userlogin:UserPassword"`

I tryed something like this but they open another shell
`runas.exe /user\:user\@domain 'powershell.exe -nologo'`

It is possible to launch somthing like this and use the vault ?
BR,

**Comment by ysc3839** [comment]
Personally I think this should be done by third-party program such as `sudo`, not by terminal.

**Reply by JGomsss** [reply]
> Personally I think this should be done by third-party program such as `sudo`, not by terminal.

Indeed, but they are no `sudo` on Windows environnement. You must open New session or use `runas.exe`
